SDKs/PowerShell/JumpCloud.SDK.V2/test/Update-JcSdkAuthenticationPolicy.Tests.ps1

```diff
@@ -16,8 +16,8 @@ Describe 'Update-JcSdkAuthenticationPolicy' {
         { Update-JcSdkAuthenticationPolicy -Id:($global:PesterTestAuthenticationPolicy.Id) [-Conditions '<Hashtable>'] [-Description '<String>'] [-Disabled] [-EffectAction '<String>'] [-MfaRequired] [-Name '<String>'] [-TargetResources '<IAuthnPolicyResourceTarget[]>'] [-Type '<String>'] [-UserGroupExclusions '<String[]>'] [-UserGroupInclusions '<String[]>'] [-UserInclusions '<String[]>'] } | Should -Not -Throw
```

Comment:
Written test should be in updateExpanded test, not update because it's missing the body param